<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="horizontal_group_box">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: bold;</string>
          </property>
          <property name="text">
           <string>Is there a datum or projection used in this Dataset  </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_yes">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Yes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_no">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>No</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="horiz_layout">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Horizontal Coordinate System</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QFrame" name="fgdc_horizsys">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="btn_geographic">
                  <property name="text">
                   <string>Geographic</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="btn_planar">
                  <property name="text">
                   <string>Planar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QStackedWidget" name="stack_definition">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="fgdc_geograph">
                 <layout class="QFormLayout" name="formLayout">
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Latitude Resolution</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="fgdc_latres"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Longitude Resolution</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="fgdc_longres"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Geographic Coordinate Units</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QComboBox" name="fgdc_geogunit">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                    <property name="currentText">
                     <string>Decimal degrees</string>
                    </property>
                    <property name="currentIndex">
                     <number>0</number>
                    </property>
                    <item>
                     <property name="text">
                      <string>Decimal degrees</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Decimal minutes</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Decimal seconds</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Degrees and decimal minutes</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Degrees, minutes, and decimal seconds</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Radians</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Grads</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="fgdc_planar">
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="leftMargin">
                   <number>3</number>
                  </property>
                  <property name="topMargin">
                   <number>3</number>
                  </property>
                  <property name="rightMargin">
                   <number>3</number>
                  </property>
                  <property name="bottomMargin">
                   <number>3</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeType">
                       <enum>QSizePolicy::Fixed</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>74</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="btn_projection">
                      <property name="text">
                       <string>Map Projection</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">btngrp_planar</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="btn_grid">
                      <property name="text">
                       <string>Grid Coordinate System</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">btngrp_planar</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="btn_local">
                      <property name="text">
                       <string>Local Planar</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">btngrp_planar</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>0</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QStackedWidget" name="stack_planar">
                    <property name="currentIndex">
                     <number>0</number>
                    </property>
                    <widget class="QWidget" name="fgdc_mapproj">
                     <layout class="QFormLayout" name="formLayout_2">
                      <property name="horizontalSpacing">
                       <number>3</number>
                      </property>
                      <property name="verticalSpacing">
                       <number>3</number>
                      </property>
                      <property name="leftMargin">
                       <number>3</number>
                      </property>
                      <property name="topMargin">
                       <number>3</number>
                      </property>
                      <property name="rightMargin">
                       <number>3</number>
                      </property>
                      <property name="bottomMargin">
                       <number>3</number>
                      </property>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>Projection Name</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QComboBox" name="fgdc_mapprojn"/>
                      </item>
                      <item row="1" column="1">
                       <widget class="QWidget" name="mapproj_contents" native="true">
                        <layout class="QFormLayout" name="formLayout_6"/>
                       </widget>
                      </item>
                      <item row="2" column="1">
                       <spacer name="verticalSpacer">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>40</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="fgdc_gridsys">
                     <layout class="QFormLayout" name="formLayout_4">
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_7">
                        <property name="text">
                         <string>Grid System Name</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QComboBox" name="fgdc_gridsysn"/>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="fgdc_local">
                     <layout class="QFormLayout" name="formLayout_3">
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>Local Planar Description</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QLineEdit" name="fgdc_localdes"/>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_6">
                        <property name="text">
                         <string>Local Planar georeference information</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QLineEdit" name="fgdc_localgeo"/>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="fgdc_geodetic">
              <property name="title">
               <string>Datum (optional)</string>
              </property>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Datum Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="fgdc_ellips">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <property name="currentIndex">
                  <number>2</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>Clarke 1866</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Geodetic Reference System 1980</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>WGS_1984</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Elipsoid Name</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Semi-major Axis</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="fgdc_semiaxis">
                 <property name="text">
                  <string>6378137.000000</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Denominator Flattening Ratio</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="fgdc_denflat">
                 <property name="text">
                  <string>298.257224</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="fgdc_horizdn">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <property name="currentIndex">
                  <number>2</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>North American Datum of 1927 (NAD 27)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>North American Datum of 1983 (NAD 83)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>World Geodetic System 1984 (WGS 84)</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btngrp_planar"/>
 </buttongroups>
</ui>
